pages/obsolete-implementations/index.page.tsx

```diff
@@ -181,13 +201,16 @@ function ImplementationTable({
                             </td>
                             <td className='w-1/4 pl-3 sm:pl-6 pb-2 pt-2'>
                               {allDrafts
-                                ?.sort((a, b) =>
-                                  DRAFT_ORDER.indexOf(a.toString()) <
-                                  DRAFT_ORDER.indexOf(b.toString())
-                                    ? -1
-                                    : 1,
+                                .sort(
+                                  (a, b) =>
+                                    DRAFT_ORDER.indexOf(
+                                      a as (typeof DRAFT_ORDER)[number],
+                                    ) -
+                                    DRAFT_ORDER.indexOf(
+                                      b as (typeof DRAFT_ORDER)[number],
+                                    ),
                                 )
```

Comment:
you can import `JSONSchemaDraft` from `lib/config` file 

```suggestion
                                .sort(
                                  (a, b) =>
                                    DRAFT_ORDER.indexOf(
                                      a.toString() as JSONSchemaDraft
                                    ) -
                                    DRAFT_ORDER.indexOf(
                                      b.toString() as JSONSchemaDraft
                                    ),
                                )
```

Reply:
Done👍